apps/portal/src/modules/media/actions/mutations.ts

```
@@ -96,14 +106,15 @@ export async function getSignedURL({ file, checksum, collection }: Props) {
- * @param media - Media object (url, fileName, fileSize, width, height, blurData, alt, userId)
- * @param transaction - Optional DB transaction
+ * Inserts a new media record into the database if one with the same URL does not exist, or returns the existing record's UUID.
+ *
+ * @param media - Media metadata including URL, file details, and user ID
+ * @param transaction - Optional database transaction to use for the operation
+ * @returns The UUID of the existing or newly inserted media record
  */
 export async function upsertMedia(
   media: MediaToInsert,
-  transaction?: any,
+  transaction?: any
```

Biome (2.1.2)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)